dotcom-rendering/src/components/FollowButtons.tsx

```diff
@@ -0,0 +1,138 @@
+import { css } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { remSpace, space, textSans } from '@guardian/source-foundations';
+import {
+	SvgCheckmark,
+	SvgNotificationsOff,
+	SvgNotificationsOn,
+	SvgPlus,
+} from '@guardian/source-react-components';
+import { palette } from '../palette';
+
+type IconProps = {
+	isFollowing?: boolean;
+	iconIsFollowing: EmotionJSX.Element;
+	iconIsNotFollowing: EmotionJSX.Element;
+};
+
+const FollowIcon = ({
+	isFollowing,
+	iconIsFollowing,
+	iconIsNotFollowing,
+}: IconProps) => (
+	<div
+		css={css`
+			display: flex;
+			margin: 0;
+			margin-right: ${space[1]}px;
+
+			border-radius: 100%;
+			border: 1px solid ${palette('--follow-icon-fill')};
+
+			svg {
+				margin: 1px;
+			}
+		`}
+		style={{
+			backgroundColor: isFollowing
+				? palette('--follow-icon-fill')
+				: undefined,
+			fill: isFollowing
+				? palette('--follow-icon-background')
+				: palette('--follow-icon-fill'),
+		}}
+	>
+		{isFollowing ? iconIsFollowing : iconIsNotFollowing}
+	</div>
+);
+
+const buttonStyles = css`
+	${textSans.small()}
+	color: ${palette('--follow-text')};
+	background: none;
+	border: none;
+	display: block;
+	margin-top: ${remSpace[1]};
+	margin-left: 0;
+	min-height: ${remSpace[6]};
```

Comment:
Space will autmatically be converted from `px` to `rem`, so we tend to use Source’s `space` directly